### ISSUE TITLE
Supported mechanisms for system-only updates?

### ISSUE THREAD

**Issue by ar-jan** [omega8cc/boa#1150]
I used to configure weekly system-ony updates via `/etc/crontab`, the initially recommended way (e.g. `18 5	* * 7	root	barracuda up-stable system`). Later the `_AUTO_UP_WEEKLY` etc. upgrade method was added and documented in [barracuda.cnf](https://github.com/omega8cc/boa/blob/master/docs/cnf/barracuda.cnf).

Is updating via (custom) cron command still supported, or is _AUTO_UP_WEEKLY the only supported way? Asking because I notice on a new BOA install (v. 3.2.0) I added an system update command in `/etc/crontab`, but it seems it is removed automatically by BOA.

Perhaps the docs in UPGRADE.txt should also point to the _AUTO_UP_WEEKLY update mechanism in .barracuda.cnf?

**Comment by omega8cc** [comment]
Indeed, `_AUTO_UP_WEEKLY` is the only supported method for a long time already, and BOA will automatically remove all other entries from `/etc/crontab`. See this script: https://github.com/omega8cc/boa/blob/master/aegir/tools/bin/autoupboa

Feel free to suggest improvements, or better yet, submit a PR :-)

BTW, you could alternatively add your custom cron entry in `/var/spool/cron/crontabs/root` and make the change permanent by adding your line to `/var/xdrago/cron/custom.txt` and then restart cron service manually.
